common/lib/dependabot/clients/azure.rb

```diff
@@ -170,7 +170,7 @@ def create_commit(branch_name, base_commit, commit_message, files,
 
       # rubocop:disable Metrics/ParameterLists
       def create_pull_request(pr_name, source_branch, target_branch,
-                              pr_description, labels, work_item = nil)
+                              pr_description, labels, work_item = nil, reviewers)
```

Comment:
I assume the linter will complain about this, but the optional argument (`work_item = nil`) should go last in the argument list, this ensures that we don't accidentally pass the wrong value when leaving it out: https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/OptionalArguments

Reply:
Make sense